src/ui/public/vislib/components/color/mapped_colors.js

```diff
@@ -1,50 +1,92 @@
-define((require) => (Private, config) => {
-  const _ = require('lodash');
-  const d3 = require('d3');
-  const createColorPalette = Private(require('ui/vislib/components/color/color_palette'));
-
-  const standardizeColor = (color) => d3.rgb(color).toString();
-  function getConfigColorMapping() {
-    return _.mapValues(config.get('visualization:colorMapping'), standardizeColor);
-  }
-
-  /*
-   * Maintains a lookup table that associates the value (key) with a hex color (value)
-   * across the visualizations.
-   * Provides functions to interact with the lookup table
-   */
-  class MappedColors {
-    constructor() {
-      this.mapping = {};
+define(function (require) {
```

Comment:
Change this back to an arrow function. This change causes all of the lines in the file to change with no functional change.